src/js/view/view.js

```diff
@@ -837,8 +837,11 @@ function View(_api, _model) {
     }
 
     function _setFloatingIntersection(entry) {
-        // Entirely invisible and no floating player already in the DOM
-        const isVisible = entry.intersectionRatio === 1;
+        // Round as the IntersectionObserver polyfill sometimes returns ±0.00XXX.
+        const intersectionRatio = Math.round(entry.intersectionRatio * 100) / 100;
```

Comment:
Can we do 1/20ths? We'll still get two decimals, but the second will always be 0 or 5 and we'll shave off two chars.